core/JsonStorage.qml

```diff
@@ -3,30 +3,47 @@ LocalStorage {
 	signal loaded;			///< json was loaded
 	signal optionUpdated;	///< option changed signal
 
-	/// load JSON method
-	load: {
-		this.read()
-		this.loaded()
+	/**
+	 * Returns JSON propeprty value by name
+	 * @param {string} name - property name
+	 * @param {function} callback - callback to return value
+	 * @param {function} error - callback to report non-existing value or some kind of error
+	 */
```

Comment:
So we will use such format for argument type specification instead of  `name:string`.
I don't get it. If so we must rewrite it everywhere and update docs generator to parse it correctly and I suppose it's another issue to solve